generator/explores/explore.py

```diff
@@ -95,6 +95,55 @@ def get_view_lookml(self, view: str) -> dict:
             return lkml.load((self.views_path / f"{view}.view.lkml").read_text())
         raise Exception("Missing view path for get_view_lookml")
 
+    def get_unnested_fields_joins_lookml(
+        self,
+    ) -> list:
+        """Get the LookML for joining unnested fields."""
+        views_lookml = self.get_view_lookml(self.views["base_view"])
+        views: List[str] = [view["name"] for view in views_lookml["views"]]
+        parent_base_name = views_lookml["views"][0]["name"]
+
+        extended_views: List[str] = []
+        if "extended_view" in self.views:
+            # check for extended views
+            extended_views_lookml = self.get_view_lookml(self.views["extended_view"])
+            extended_views = [view["name"] for view in extended_views_lookml["views"]]
+
+            views_lookml.update(extended_views_lookml)
+            views += extended_views
```

Comment:
Append seems a little clearer to me here

```suggestion
            views.append(extended_views)
```

Reply:
`append()` won't work here as it would not merge the two lists, but append `extended_views` as a single item:

```python
views = ['a', 'b', 'c']
extended = ['d', 'e']

views += extended  # -> views = ['a', 'b', 'c', 'd', 'e']

views.append(extended) # -> views = ['a', 'b', 'c', ['d', 'e']]
```

Reply:
Oh, right. In that case, I think "extend" might have been better. No worries though